backends/pipeline_api.py

```diff
@@ -102,6 +90,30 @@ def update_profile_from_edx(backend, user, response, is_new, *args, **kwargs):
     )
 
 
+def check_verified_email(backend, response, details, *args, **kwargs):  # pylint: disable=unused-argument
+    """Get account information to check if email was verified for account on edX"""
+    if backend.name != EdxOrgOAuth2.name:
+        return
```

Comment:
Suggest you return `{}` here, I've found it's best to be consistent in always returning some kind of dict out of PSA actions in case you change it to a partial later.

Reply:
Looks like `pylint` caught this anyway.